postgres/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 <!-- towncrier release notes start -->
 
-## 22.7.0 / 2025-03-19
+## 22.7.0 / 2025-03-19 / Agent 7.64.0
```

Comment:
Actually - should this be 7.64.0 or 7.64.2?

Reply:
I think it did indeed comeout with 7.64.0
https://github.com/DataDog/integrations-core/blob/7.64.0/postgres/CHANGELOG.md#2270--2025-02-26